tests/forms/test_user.py

```diff
@@ -11,84 +11,87 @@
 
 
 # noinspection PyUnusedLocal
-def test_register_form_validate_without_full_name(db):
+def test_register_form_validate_without_full_name(superuser, db):
     """Attempt registering user with name shorter than 3 chars."""
-    form = RegisterForm(username='mr.librarian@kb.se', full_name='01',
+    form = RegisterForm(superuser, current_user=superuser, username='mr.librarian@kb.se',
```

Comment:
I think there is no `current_user` kwarg for *user.forms.RegisterForm*. WTForms is very forgiving... I think you could easily pass in `code_monkey=True` and it wouldn't say a word about it. Removing it?